python/cudf/cudf/core/udf/masked_typing.py

```diff
@@ -34,7 +41,46 @@
     types.NPTimedelta,
 )
 
+SUPPORTED_NUMPY_TYPES = (
+    NUMERIC_TYPES | DATETIME_TYPES | TIMEDELTA_TYPES | STRING_TYPES
+)
+supported_type_str = "\n".join(sorted(list(SUPPORTED_NUMPY_TYPES) + ["bool"]))
+MASKED_INIT_MAP: Dict[Any, Any] = {}
+
+
+def _format_error_string(err):
```

Comment:
This function should indicate the meaning of the escape symbols (red text) in a docstring (`"""Output an error code on a newline in red text."""`) or variable names like `red_start = "\033[91m"`.

Reply:
I updated this.